notebooks/04_model_select_and_optimize/tuning_spark_als.ipynb

```diff
@@ -37,16 +37,16 @@
   },
```

Comment:
<p>We should control these via papermill parameters too and we can adjust the number of options for rank and reg in the test,  I don't think we really need to evaluate multiple options in the unit test.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/864/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Yea the subset data size is controlled by a parameter that can be modified by papermill in testing.